gemma.cc

```diff
@@ -1090,5 +1235,16 @@ void CompressWeights(gcpp::Model model, const Path& weights,
   (model, weights, compressed_weights, pool);
 }
 
+float ComputeCrossEntropy(Gemma& gemma, size_t max_tokens,
+                          const std::vector<int>& prompt, KVCache& kv_cache,
+                          hwy::ThreadPool& pool, hwy::ThreadPool& inner_pool,
+                          int verbosity) {
+  pool.SetWaitMode(hwy::PoolWaitMode::kSpin);
```

Comment:
Is this time-critical? We can stick with the default/current wait mode, I think.

Reply:
It's part of a benchmark, which I assume why this is here :-)